code/modules/food&drinks/drinks/drinks/bottle.dm

```diff
@@ -33,7 +33,7 @@
 	qdel(src)
 
 /obj/item/weapon/reagent_containers/food/drinks/bottle/attack(mob/living/target as mob, mob/living/user as mob)
-
+	..()
```

Comment:
This is going to make breaking bottles on someone's head impossible. Why are you adding this?